spec/services/record_declarations/retained/early_career_teacher_spec.rb

```diff
@@ -1,46 +1,12 @@
 # frozen_string_literal: true
 
 require "rails_helper"
+require "./rspec/shared/context/service_record_declaration_params.rb"
+require "./rspec/shared/context/leadprovider_profiles_and_courses.rb"
```

Comment:
Let's name it `lead_provider` rather than `leadprovider` just to keep consistent across codebase.